tests/unit/components/atoms/EmptyTodo.spec.js

```diff
@@ -0,0 +1,13 @@
+import { describe, expect, it } from '@jest/globals'
+import { shallowMount } from '@vue/test-utils'
+import EmptyTodo from '@/components/atoms/EmptyTodo.vue'
+
+describe('Test EmptyTodo', () => {
+  it('should render', () => {
+    const wrapper = shallowMount(EmptyTodo, {})
+
+    expect(wrapper.exists()).toBe(true)
+    expect(wrapper.find('span').element.textContent).toBe('No Todo yet')
+    expect(wrapper.element).toMatchSnapshot()
```

Comment:
Usually the snapshot is checked first.
There is also no need to check wrapper.exists and textContent, all of this will be checked in the snapshot